hieradata/common.yaml

```diff
@@ -1638,6 +1638,7 @@ router::assets_origin::app_specific_asset_routes:
   '/smartanswers/': "smartanswers"
 
 router::assets_origin::asset_manager_routes:
+  - '/media/'
```

Comment:
I think it might be worth mentioning in the commit note that you're doing this to DRY up `assets_origin.conf.erb`.

Reply:
I've added the following to the commit note:

> In preparation for being able to reuse the `router::assets_origin::asset_manager_routes` array in the nginx config file for the static virtual host.